src/js/gpio.js

```diff
@@ -100,7 +100,7 @@ function GpioPin(configuration, callback) {
   })(this));
 }
 
-// gpio.write(value[, callback])
+// gpiopin.write(value[, callback])
```

Comment:
I don't think these comments give more information. What about getting rid of these comments?